assets/js/pages/SettingsPage/SettingsPage.jsx

```diff
@@ -263,6 +277,8 @@ function SettingsPage() {
         </div>
       </div>
       <ApiKeySettingsModal
+        userAbilities={abilities}
```

Comment:
As carmine said we can remove these. We don't need to protect anything within the modal